src/main/java/com/cjcrafter/armormechanics/durability/DurabilityPlaceholderHandler.kt

```diff
@@ -0,0 +1,21 @@
+package com.cjcrafter.armormechanics.durability
+
+import me.deecaad.core.placeholder.NumericPlaceholderHandler
+import me.deecaad.core.placeholder.PlaceholderData
+import me.deecaad.core.placeholder.PlaceholderHandler
+
+fun registerDurabilityPlaceholders() {
+    PlaceholderHandler.REGISTRY.add(object : DurabilityPlaceholderHandler() {})
+    PlaceholderHandler.REGISTRY.add(object : MaxDurabilityPlaceholderHandler() {})
+}
+abstract class DurabilityPlaceholderHandler : NumericPlaceholderHandler("durability_current") {
```

Comment:
Why are these abstract?

Reply:
Used "AttributePlaceholderHandler" as modal and it is abstract but can make it not abstract.